generic_sample.txt

```diff
@@ -12,7 +12,7 @@ START SCENE:
 		choice:
 			text: "this is a start choice"
 			next scene: test 2
-STORY SCENE:
+STORY SCENES:
```

Comment:
Unsure whether we should just call everything SCENE or SCENES. There's a small disparity by writing START SCENE and eg. STORY SCENES. Not sure if this is confusing, but I'm using both for now